earthfile2llb/converter.go

```diff
@@ -1536,6 +1579,14 @@ func (c *Converter) internalRun(ctx context.Context, opts ConvertRunOpts) (pllb.
 		opts.shellWrap = withShellAndEnvVars
 	}
 
+	if c.ftrs.WaitBlock {
+		c.opt.Console.Warnf("WAIT/END code is super-experimental and incomplete -- it should currently be avoided")
```

Comment:
Heh... "super-experimental" is maybe too much. Just "experimental" is enough? Perhaps we can say that it should be avoided for anything other than toy projects.

```suggestion
		c.opt.Console.Warnf("WAIT/END code is super-experimental and incomplete -- it should currently be avoided")
```